### DISCUSSION TITLE
Nothing render on screen

### DISCUSSION THREAD

**Discussion by alexlaw2006** [xyflow/xyflow#2317] [Q&A]
Here is the dependencies in package,json
"dependencies": {
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-flow-renderer": "^10.3.12",
    "react-scripts": "5.0.1"
  },

If I use version 9.7.4, the chart could be rendered but I cannot use the functions {useNodesState, useEdgesState}
export 'useNodesState' (imported as 'useNodesState') was not found in 'react-flow-renderer'
export 'useEdgesState' (imported as 'useEdgesState') was not found in 'react-flow-renderer'

**Comment by moklick** [comment]
Hey! v9 doesn't have `useNodesState` or `useEdgesState`. What is the issue with v10? Do you get any console logs? Could you create a codesandbox?

**Reply by alexlaw2006** [reply]
Hello moklick. Here is the codesandbox.
https://codesandbox.io/s/gallant-wildflower-cup0zf?file=/src/App.js

**Reply by alexlaw2006** [reply]
my reference. 
https://reactflow.dev/docs/examples/interaction/drag-and-drop/

**Reply by moklick** [reply]
Something is wrong with the CSS. You need to remove the `height` of the `.react-flow-wrapper`:

```css
.dndflow .reactflow-wrapper {
  flex-grow: 1;
}
```

**Reply by alexlaw2006** [reply]
Thank you so much :)
